lib/modules/hosts/simplecove.js

```diff
@@ -0,0 +1,28 @@
+import { ajax } from '../../environment';
+
+export default {
+	moduleID: 'simplecove',
+	name: 'SimpleCove',
+	domains: ['simplecove.com'],
+	logo: 'https://simplecove.com/static/images/reslogo.jpg',
+	detect: ({ href }) => (/^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)(?:\/)?.+/g).exec(href),
```

Comment:
You are correct. I changed it to ^https?:\/\/(?:www\.)?simplecove\.com\/(\d+) . Thank you.